angel.py

```diff
@@ -161,42 +174,31 @@ def modify_player_list(player_list):
                 player.genderpref = GENDER_MALE
 
 
-def write_to_csv(index, name01, *player_lists):
+def write_to_csv(index, name01, column_names, *player_lists):
     '''
     Writes a variable number of player lists to csv
     '''
     for player_list in player_lists:
         if player_list is not None:
             print(f"Length of list: {len(player_list)}")
             cur_time = time.strftime("%Y-%m-%d %H-%M-%S")
-            with open(f"{index} - {name01} - {cur_time}.csv", 'w', newline='') as f: # In Python 3, if do not put newline='' AND choose 'w' instead of 'wb', you will have an empty 2nd row in output .csv file.
+            with open(f"{index} - {name01} - {cur_time}.csv", 'w',
+                      newline='') as f:  # In Python 3, if do not put newline='' AND choose 'w' instead of 'wb', you will have an empty 2nd row in output .csv file.
```

Comment:
I would put the code as a single line, then put the comment in two subsequent lines. Though i'm not clear whether what I proposed is standard practise